lib/taskana-core/src/main/java/pro/taskana/task/internal/ObjectReferenceMapper.java

```diff
@@ -20,6 +20,7 @@ public interface ObjectReferenceMapper {
       "<script>SELECT ID, COMPANY, SYSTEM, SYSTEM_INSTANCE, TYPE, VALUE "
           + "FROM OBJECT_REFERENCE "
           + "WHERE ID = #{id}"
+          + "ORDER BY COMPANY ASC, SYSTEM ASC, SYSTEM_INSTANCE ASC, TYPE ASC, VALUE ASC"
```

Comment:
This does not match with the new order in the indeces

Reply:
i forgot this...


Done